services/frontend-service/src/ui/utils/store.tsx

```diff
@@ -966,25 +968,37 @@ export type GlobalLoadingState = {
     configReady: boolean;
     isAuthenticated: boolean;
     azureAuthEnabled: boolean;
+    dexAuthEnabled: boolean;
     overviewLoaded: boolean;
 };
 
 // returns one loading state for all the calls done on startup, in order to render a spinner with details
-export const useGlobalLoadingState = (): [boolean, GlobalLoadingState] => {
+export const useGlobalLoadingState = (): React.ReactElement | undefined => {
     const { configs, configReady } = useFrontendConfig((c) => c);
     const isAuthenticated = useIsAuthenticated();
     const azureAuthEnabled = configs.authConfig?.azureAuth?.enabled || false;
+    const dexAuthEnabled = configs.authConfig?.dexAuth?.enabled || false;
     const overviewLoaded = useOverviewLoaded();
```

Comment:
the overview can only be loaded once authenticated. So we first need to check that the token is valid before checking the overview